portals/publisher/src/main/webapp/source/src/app/components/Apis/Listing/TableView/TableView.jsx

```diff
@@ -427,6 +407,39 @@ class TableView extends React.Component {
                     sort: false,
                 },
             },
+            {
+                name: 'type',
+                label: intl.formatMessage({
+                    id: 'Apis.Listing.ApiTableView.type',
+                    defaultMessage: 'Type',
+                }),
+                options: {
+                    customBodyRender: (value, tableMeta, updateValue, tableViewObj = this) => {
+                        const apiData = tableViewObj.state.apisAndApiProducts?.[tableMeta.rowIndex];
+                        if (apiData) {
+                            if (apiData.subtype && apiData.subtype === 'AIAPI') {
+                                return 'AI API';
+                            } else if (apiData.type === 'MCP') {
+                                return 'MCP Server';
+                            } else if (apiData.type === 'APIPRODUCT') {
```

Comment:
Do we have constants for these values? If so shall we use them here instead hardcoding?

Reply:
Added the constants with 5f90929.